### PR TITLE
refactor: remove isService filter and svc expr substitution

### PR DESCRIPTION
**Pull request by gitlawr**
**Related Issue:**

https://github.com/seal-io/walrus/issues/1770

### CODE REVIEW

File: pkg/apis/dashboard/extension_view.go [hunk 1]
```diff
@@ -28,8 +28,6 @@ type (
 	CollectionRouteGetLatestResourceRevisionsRequest struct {
```
Comment (alexcodelf) [inline pkg/apis/dashboard/extension_view.go:28]: `CollectionRouteGetBasicInformationResponse`'s attributes contains Service

File: pkg/apis/dashboard/extension.go
Comment (alexcodelf) [inline pkg/apis/dashboard/extension.go:1]: `func (h Handler) CollectionRouteGetBasicInformation` will select serviceNum

**Review by alexcodelf — APPROVED** [review]
LGTM

**Comment by gitlawr** [comment]
Rebased

**Review by thxCode — APPROVED** [review]
LGTM

**Pull request merged**